IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerTokenRequest.m

```diff
@@ -22,20 +22,34 @@
 // THE SOFTWARE.
 
 #import "MSIDDefaultBrokerTokenRequest.h"
+#import "MSIDInteractiveRequestParameters.h"
+#import "MSIDAccountIdentifier.h"
 
 @implementation MSIDDefaultBrokerTokenRequest
 
 // Those parameters will be different depending on the broker protocol version
 - (NSDictionary *)protocolPayloadContentsWithError:(NSError **)error
 {
-    // TODO: MSAL pieces
-    return @{};
+    NSString *homeAccountId = self.requestParameters.accountIdentifier.homeAccountId;
+    NSString *username = self.requestParameters.accountIdentifier.legacyAccountId;
+    
+    NSDictionary *contents =
+    @{
+      @"scope": self.requestParameters.target ?: @"",
```

Comment:
Question for future extensibility: Do we want broker be responsible for appending OIDC scopes, or we want MSAL to drive OIDC scopes requested?

Reply:
Talked to Olga offline. Client may want additional OIDC scopes, so we let client send the OIDC scopes. In broker, we will also check if the scope contains minimal OIDC scopes, add them if not.